nepalingo-web/src/hooks/StreakContext.tsx

```diff
@@ -60,8 +60,8 @@ export const StreakProvider = ({ children }: { children: ReactNode }) => {
     const currentDate = new Date().toISOString().split("T")[0]; // Get current date
 
     if (user) {
-      let newStreak = 0;
-      let newLongestStreak = 0;
+      let newStreak = 1;
+      let newLongestStreak = 1;
```

Comment:
Streaks already default to value 0 when no rows exist. This function is only called when the flashcard page is loaded, so at this point the streaks should actually be updated to 1